src/events.js

```diff
@@ -1919,19 +1940,58 @@ export const goToParticipantSearch = () => {
     document.getElementById('navBarSearch').click();
 }
 
-export const addEventSpecimenLinkForm = (formData) => {
+export const addEventSpecimenLinkForm = (formData, visitConcept) => {
```

Comment:
visitConcept not being used, needs to be removed.

Reply:
It's being used in line 1930